cmake/extensions.cmake

```diff
@@ -389,6 +389,19 @@ macro(zephyr_library)
   zephyr_library_named(${lib_name})
 endmacro()
 
+# Checks if a Kconfig is enabled and then does the same operation as
```

Comment:
The documentation should not mention Kconfig as there is nothing Kconfig-specific about this function.